Modules/rat_rescue.py

```diff
@@ -770,11 +771,13 @@ def rats(self, value):
 
             ```
         """
+        found_rat: Rat = None
+
         if isinstance(rat, Rat):
             # we already have a rat object, lets verify it has an ID and assign it.
             if rat.uuid is not None:
                 self.rats.append(rat)
-                return
+                found_rat = rat
```

Comment:
`found_rat` is a bit of a misnomer in this context, as the `Rat` was provided as a argument. may i suggest renaming this variable `assigned_rat`?

Reply:
Yeah, will change.